docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -2147,3 +2149,143 @@ function sitenow_update_9046() {
     $block->save();
   });
 }
+
+/**
+ * Update several fields from plain text to formatted basic text.
+ */
+function sitenow_update_9047() {
+
+  // Statically defining our fields to be updated. More can be added here.
+  $fields = [
+    'field_uiowa_quote_content' => [
+      'format_col' => 'field_uiowa_quote_content_format',
+    ],
+    'field_uiowa_quote_footer' => [
+      'format_col' => 'field_uiowa_quote_footer_format',
+    ],
+  ];
+  // Get the field and revision storage table names for each field.
+  $table_mapping = \Drupal::service('entity_type.manager')
+    ->getStorage('block_content')
+    ->getTableMapping();
+  $storage_definitions = \Drupal::service('entity_field.manager')
+    ->getActiveFieldStorageDefinitions('block_content');
+  foreach ($fields as $field_name => &$field) {
+    $field_definition = $storage_definitions[$field_name];
+    if ($field_definition) {
+      $field['table'] = $table_mapping->getDedicatedDataTableName($field_definition);
+      strip_tags($field['revision_table'] = $table_mapping->getDedicatedRevisionTableName($field_definition));
```

Comment:
Not sure what `strip_tags` does since it is fetching a table name

Reply:
Adding it there removed some `<p>` tags that were being rendered around the field.   I'm open to other suggestions on how to strip the tags from the revisions.

Reply:
>Adding it there removed some <p> tags that were being rendered around the field.

i just don't understand how this does this. `$table_mapping->getDedicatedRevisionTableName($field_definition)` returns a table name string.

<img width="838" alt="Screen Shot 2022-08-18 at 4 01 00 PM" src="https://user-images.githubusercontent.com/4663676/185494061-395ac445-f401-4776-8b43-57f8d8e8e599.png">

I haven't traced down that example of `<p>` tags being included but I would imagine stripping the tags on the data $rows and $revision_rows would be where this happens.

<img width="495" alt="Screen Shot 2022-08-18 at 4 02 28 PM" src="https://user-images.githubusercontent.com/4663676/185494276-1609b964-b43a-4330-9f3e-68a2474293a3.png">

Reply:
Either way, I would think that strip tags should go around whatever is being evaluated on the right side of the `=`. If for example the `=` condition were to evaluate to `FALSE`, then I'm not sure this is something that `strip_tags` could handle.

Reply:
I looked for the `<p>` tag in xdebug too but couldn't find it there and just saw the string.   I moved the `strip_tags` to the right side and it is still working.

Reply:
I might need to have you run through this in a huddle tomorrow. I just removed strip_tags from the update hook and synced the sandbox site. I am not seeing `<p>` tags either.

Also if this is doing what it says it does, why is only the revision_table addressed? Shouldn't we do this for table too?

Reply:
Sounds good. Let's huddle after standup if you are available.